[2024-06-24T00:00:01.884+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T00:00:01.904+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_to_s3.fetch_nasa_data scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T00:00:01.910+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_to_s3.fetch_nasa_data scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T00:00:01.910+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-24T00:00:01.922+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_nasa_data> on 2024-06-23 00:00:00+00:00
[2024-06-24T00:00:01.929+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1632) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T00:00:01.931+0000] {standard_task_runner.py:63} INFO - Started process 1634 to run task
[2024-06-24T00:00:01.930+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_to_s3', 'fetch_nasa_data', 'scheduled__2024-06-23T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/extractAPIdata.py', '--cfg-path', '/tmp/tmpsxsq0vix']
[2024-06-24T00:00:01.933+0000] {standard_task_runner.py:91} INFO - Job 36: Subtask fetch_nasa_data
[2024-06-24T00:00:01.973+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_to_s3.fetch_nasa_data scheduled__2024-06-23T00:00:00+00:00 [running]> on host e0fa33b4439a
[2024-06-24T00:00:02.051+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_to_s3' AIRFLOW_CTX_TASK_ID='fetch_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-23T00:00:00+00:00'
[2024-06-24T00:00:02.053+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T00:00:03.137+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-24T00:00:03.138+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T00:00:03.148+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_to_s3, task_id=fetch_nasa_data, run_id=scheduled__2024-06-23T00:00:00+00:00, execution_date=20240623T000000, start_date=20240624T000001, end_date=20240624T000003
[2024-06-24T00:00:03.188+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T00:00:03.211+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-24T00:00:03.212+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
